birdhouse/config/jupyterhub/default.env

```diff
@@ -5,7 +5,7 @@
 # are applied and must be added to the list of DELAYED_EVAL.
 
 export JUPYTERHUB_DOCKER=pavics/jupyterhub
-export JUPYTERHUB_VERSION=1.4.0-20210506
+export JUPYTERHUB_VERSION=3.0.0-20220926
```

Comment:
Just to be sure, are the existing images that used the older JupyterHub version still able to run using this newer version?

Reply:
Yes jupyterLab works out of the box with JupyterHub 1.0 +